src/main/kotlin/Graph.kt

```diff
@@ -0,0 +1,53 @@
+import Graph.Companion.STOCKHOLM
+import Graph.Companion.UPPSALA
+
+
+class Graph {
+    companion object {
+        const val STOCKHOLM = "ID_STOCKHOLM"
+        const val UPPSALA = "ID_UPPSALA"
+    }
+
+    val user: User? by lazy {
+        User()
+    }
+
+    fun findShortestPath(node1: Node, node2: Node): Int {
+        return 0
+    }
+
+    fun getNodeById(id: String): Node {
+
+        val firstName = user!!.firstName
+
+        return Node()
+    }
+}
+
+class Node {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tokgozmusa/detekt-danger/blob/ca0557522d0aad924a51938f9c9a3ee21221af3d/src/main/kotlin/Graph.kt#L27"></span>The class or object Node is empty.</td>
    </tr>
  </tbody>
</table>